easybuild/easyconfigs/p/plotly.py/plotly.py-4.4.1-intel-2019b.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonBundle'
+
+name = 'plotly.py'
+version = '4.4.1'
+
+homepage = 'https://plot.ly/python'
+description = "An open-source, interactive graphing library for Python"
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+multi_deps = {'Python': ['3.7.4', '2.7.16']}
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('retrying', '1.3.3', {
+        'checksums': ['08c039560a6da2fe4f2c426d0766e284d3b736e355f8dd24b37367b0bb41973b'],
+    }),
+    ('plotly', version, {
```

Comment:
@boegel for consistency, use `name` here?

Reply:
Changing that was a mistake, see #9599...